t/page/house_wikidata.rb

```diff
@@ -0,0 +1,22 @@
+require 'minitest/autorun'
+require_relative '../../lib/page/house_wikidata'
+
+describe 'HouseWikidata' do
+  subject do
+    Page::HouseWikidata.new('austria', 'Nationalrat')
```

Comment:
It's generally best to make the primary test be the way that something is actually used — i.e. 'nationalrat', rather than 'Nationalrat'. (If you need to test that it's case-insensitive, then you can add a second test that explicitly says that's what it's doing, but I don't think you need that here)